### PR TITLE
Improve docs for `evaluate` and `evaluateAsync`

### PR DESCRIPTION
**Pull request by sqd**

### CODE REVIEW

File: _posts/api/webpage/method/2000-01-01-evaluate.md [hunk 1]
```diff
@@ -52,8 +52,28 @@ page.open('http://m.bing.com', function(status) {
...
+var webPage = require('webpage');
+var page = webPage.create();
+
+page.onConsoleMessage = function(msg) {
+  console.log(msg);
+}
+
+page.open('http://m.bing.com', function(status) {
+
+  var title = page.evaluate(function() {
```
Comment (Vitallium) [inline _posts/api/webpage/method/2000-01-01-evaluate.md]: we can omit defining variable `title` here

**Comment by zackw** [comment]
@Vitallium Do you see any other issues with this?  If not, I can make your suggested change and merge it.

**Comment by Vitallium** [comment]
@zackw I don't see anything else.

**Comment by zackw** [comment]
Merged.  @sqd Thank you for the contribution.

**Pull request closed without merging**